retrieval_service/datastore/providers/cloudsql_postgres_test.py

```diff
@@ -330,7 +330,7 @@ async def test_get_amenity(ds: cloudsql_postgres.Client):
         saturday_end_hour=None,
     )
     assert res == expected
-    assert sql is None
+    assert sql is not None
```

Comment:
This doesn't seem like a particularly useful check -- we should either remove it or assert the SQL is something expected